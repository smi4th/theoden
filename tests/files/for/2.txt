a=0;
for(i=10;i>0;i=i-1){
    a=a+1;
}