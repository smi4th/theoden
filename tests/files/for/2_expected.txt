---TEST OUTPUT---
---MEMORY---
a = ('int', 10)
---FUNCTIONS---
